### ISSUE TITLE
Is it possible to remove autogenerated styles by gatsby-background-image?

### ISSUE THREAD

**Issue by conspireagency** [timhagn/gatsby-background-image#107]
## Summary
It seems that the BackgroundImage component adds autogenerated styles for the component, both at the div it generates and as inline styles.  In this example I'm passing styles to the component for background position, but something else entirely is being added as an inline style to the main div.  The inline styles are something else completely, with odd properties like background-position-x that override background-position settings.

Is there something I'm missing?

## Relevant information

`<BackgroundImage
                className="c-project__img"
                fluid={data.bodybioCase.childImageSharp.fluid}
                style={{
                  position: "absolute",
                  backgroundPosition: "100% 33%",
                  backgroundRepeat: "none",
                  backgroundSize: "55%"
                }}
              ></BackgroundImage>`

Div output 

`<div class="c-project__img gbi-961338430-pStG3MMasNkjkYBgtAWBV9 gatsby-image-wrapper" style="position: absolute; background-size: cover; background-position: 50% 50%; opacity: 0.99; background-repeat: no-repeat;">`

```
.c-project__img.gbi-961338430-pStG3MMasNkjkYBgtAWBV9:before, .c-project__img.gbi-961338430-pStG3MMasNkjkYBgtAWBV9:after {
	content: '';
	display: block;
	position: absolute;
	width: 100%;
	height: 100%;
	top: 0;
	left: 0;
	transition: none;
	background-position: 100% 75%;
	background-repeat: no-repeat;
	background-size: 60%;
	background-position-x: center;
	background-position-y: center;
	background-repeat-x: no-repeat;
	background-repeat-y: no-repeat;
}
```

### File contents (if changed)

`package.json`: N/A

**Comment by timhagn** [comment]
Hi @conspireagency,

sorry for responding so late, had two quite stressful weeks, as one might guess % ).
Usually the styles put in should trump the default styling, so I'm quite astounded by your findings.
Gonna try digging in this week : ).

Best,

Tim.

**Comment by timhagn** [comment]
Hi @conspireagency,

this should now be fixed with the just released `gbi` `v1.0.0` : ).
(Oh,  and btw. it's `backgroundRepeat: 'no-repeat'`, not none shouldn't you meanwhile seen it yourself.)
When you're happy with the result, you of course have the honor to close this ; ).

Best,

Tim.
